tests/data/scripts/mssql.sh

```diff
@@ -5,7 +5,7 @@
 
 for i in {1..50};
 do
-    /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P $SA_PASSWORD -d master -Q "CREATE DATABASE $MSSQL_DATABASE;"
+    /opt/mssql-tools18/bin/sqlcmd -S localhost -U sa -P $SA_PASSWORD -No -d master -Q "CREATE DATABASE $MSSQL_DATABASE;"
```

Comment:
@iliayatsenko Do you have some reference about this renaming?

Could this be the reason for the failing pipeline in https://github.com/Codeception/module-db/pull/77#issuecomment-2522450830 as well?

Reply:
@W0rma , yes, all description is in related issue. In short, please see this comment - https://github.com/microsoft/mssql-docker/issues/892#issuecomment-2253179915

Regarding #77 , yes, it looks like the reason is the same.

Reply:
I suggest replacing the MSSQL image with server:2022-CU13-ubuntu-20.04, as the path issue is fixed in this version. This change will help avoid breaking the CI again if they revert the path in the future.

You can replace the image in [docker_compose.yaml](https://github.com/Codeception/module-db/blob/master/docker-compose.yml) and [github action configuration](https://github.com/Codeception/module-db/blob/master/.github/workflows/main.yml).

I've tested it on my branch and local machine, and it works properly.